supplemental-ui/.htaccess

```diff
@@ -1,25 +1,10 @@
 DirectoryIndex index.html index.asciidoc
 
-# former path to antora path
-Redirect 301 /about/ /front/main/about/
```

Comment:
The current page in our website is https://netbeans.apache.org/about/. 
All search engines have already indexed that page, so when a user searchs Google and is directed to the "/about/" page we want to behave nicely to them, this is, we do want to redirect them to "/front/main/about". So I would keep this one.